recipe/meta.yaml

```diff
@@ -1,43 +1,53 @@
-{% set version = "1.5.0" %}
+{% set name = "geoviews" %}
+{% set version = "1.5.1" %}
 
 package:
-  name: geoviews
+  name: {{ name }}
   version: {{ version }}
 
-source:
-  url: https://pypi.io/packages/source/g/geoviews/geoviews-{{ version }}.zip
-  sha256: e904010fc8517f23fd9faaa6a8957c1626e78f1447494d193d80547c837d3cba
-
 build:
   number: 0
   noarch: python
-  script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
   build:
     - python
     - pip
     - param >=1.6.1,<2.0
   run:
-    - python
-    - param >=1.6.1,<2.0
-    - numpy >=1.0
-    - holoviews >=1.10.0
-    - cartopy >=0.14.2
-    # extras
+    - python >=2.7
+    - datashader
+    - gdal
+    - geopandas
+    - geoviews-core ={{ version }}
     - jupyter
-    - notebook
-    - bokeh >=0.12.15
+    - libgdal
+    - matplotlib
+    - netcdf4 <1.4.0
```

Comment:
This is requiring a lot of extras! Indeed a `core` package is needed for this case.
Are you planning on using `split` packages and a single recipe?

Reply:
I'm not planning to do anything different from what I've already done here and at https://github.com/conda-forge/staged-recipes/pull/6042 (i.e. geoviews is a metapackage that depends on geoviews-core, and the recipes are separate`(*)`). However, I'm totally open to suggestions about how to do it better (especially if you can link to an example I can copy...).
(I should point out I'm not an author or maintainer of geoviews; any opinions I express are my own :) )

`(*)` we release (dev) packages to the pyviz (dev) channel; those recipes are templated to avoid duplication of metadata, which also ensures the version of geoviews and geoviews-core matches.